### ISSUE TITLE
[Select]: a11y  collapsed Select does not have the role of collpased

### ISSUE THREAD

**Issue by AlexYanSap** [SAP/ui5-webcomponents#9332]
### Bug Description

It's an incident from the customer that the collapsed select should have the role of the collapsed as recommended by the https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.

I noticed the SR did announce the collapsed state even though we have not set the role as recommended.

![image](https://github.com/SAP/ui5-webcomponents/assets/169127980/26bcef26-f65a-402f-9e21-55960a33816d)

### Affected Component

Select

### Expected Behaviour

Ensure role should be button collapsed. 
![image](https://github.com/SAP/ui5-webcomponents/assets/169127980/998b1dde-ed16-4bc9-95e7-a5358acc9afe)

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/Select/

### Steps to Reproduce

1. use Edge (123.0.2420.81 (Official build) (64-bit))
2. focus on any of the select on the Isolated Example Page
3. no collapsed role is set on the Select

### Priority

None

### UI5 Web Components Version

1.24.1

### Browser

Edge

### Additional Context

I'm from SFSF

### Organization

SAP SF

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by kineticjs** [comment]
Hi @AlexYanSap,

Can we ask the customer to give more details i.e. indicate more precisely their concern.

The spec https://www.w3.org/TR/wai-aria/#widget_roles does not list any role named "collapsed". 

1. "Collapsed" is rather a state and it is indicated with the `aria-expanded` attribute: https://www.w3.org/TR/wai-aria/#aria-expanded The ui5-select currently updates that attribute whenever the dropdown in shown/hidden, so the SR announces the new state
2. By spec, the role should be `button` only if the select is a button that opens a dropdown, otherwise (as is the case of ui5-select) it should be combobox/listbox . Currently the ui5-select has rile combobox https://www.w3.org/TR/wai-aria/#combobox
